testing/simapp/simd/cmd/root.go

```diff
@@ -164,7 +165,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		genutilcli.ValidateGenesisCmd(simapp.ModuleBasics),
 		AddGenesisAccountCmd(simapp.DefaultNodeHome),
 		tmcli.NewCompletionCmd(rootCmd, true),
-		testnetCmd(simapp.ModuleBasics, banktypes.GenesisBalancesIterator{}),
+		cmd.NewTestnetCmd(simapp.ModuleBasics, banktypes.GenesisBalancesIterator{}),
```

Comment:
I removed our testnet.go file and import instead the one from the SDK. This allows us to run a 4-validator local testnet like is described [here](https://docs.cosmos.network/v0.45/run-node/run-testnet.html).